doc/Protocol.md

```diff
@@ -0,0 +1,39 @@
+Used Protocols
+===============
+
+Friendicas DFRN Protocol
+---
+
+* [Document with the DFRN specification](spec/dfrn2.pdf)
+* [Schema of the contact request process](spec/dfrn2_contact_request.png)
+* [Schema of the contact request confirmation](spec/dfrn2_contact_confirmation.png)
+* [Description of the message flow](help/Message-Flow)
+
+ActivityStreams
+---
+
+Friendica is using ActivityStreams in version 1.0 for its activities and object types. Additional types are using for non standard activities.
```

Comment:
Can we write this rules down somewhere? It's hard to remember the docu rules when someone don't work very often at the documentation part